apps/trading/components/welcome-dialog/welcome-notice-dialog.tsx

```diff
@@ -28,8 +26,7 @@ export const WelcomeNoticeDialog = () => {
       </h1>
       <p className="leading-6 mb-7">
         {t(
-          'Vega %s is now live, but markets need to be voted for before the can be traded on. In the meantime:',
-          [networkName]
+          'Trading on Vega is now live, but markets need to be voted for before they can be traded on. In the meantime:'
         )}
       </p>
       <ul className="list-[square] pl-7">
```

Comment:
Below, could you change the link text from 'Propose your own markets' to "Propose a market"?